sys-devel/llvm/llvm-16.0.0_rc2.ebuild

```
@@ -325,6 +325,10 @@ get_distribution_components() {
 }
 
 multilib_src_configure() {
+	#GCC -flto ICE protection #873670
```

Please adjust all to:
```suggestion
	# GCC -flto miscompiles LLVM, bug #873670
```